drivers/sensors/cirque_pinnacle.c

```diff
@@ -331,6 +331,15 @@ pinnacle_data_t cirque_pinnacle_read_data(void) {
     result.wheelCount = ((int8_t*)data)[3];
 #endif
 
+#ifdef CIRQUE_PINNACLE_REACHABLE_CALIBRATION
+    static uint16_t xMin = 0xffff, yMin = 0xffff, yMax = 0, xMax = 0;
```

Comment:
```suggestion
    static uint16_t xMin = UINT16_MAX, yMin = UINT16_MAX, yMax = 0, xMax = 0;
```

Nit, the maximum values of all fixed width integer types are available as macro constants IMHO we should use them for readability.